containers/utils/uploaded.py

```diff
@@ -116,11 +123,11 @@ def __init__(self, config_parser):
         WIDTH, HEIGHT = camera_config["width"], camera_config["height"]
         # print(WIDTH, HEIGHT, "DIMS")
         camera.setResolution(WIDTH, HEIGHT)
-        camera.setExposureManual(50)
+        #camera.setExposureManual(50)
         self.cvSink = cs.getVideo()
         self.img = np.zeros(shape=(HEIGHT, WIDTH, 3), dtype=np.uint8)
         self.output = cs.putVideo("Axon", WIDTH, HEIGHT)
-
+        print("Using Port Number " + str(cs.kBasePort + 1))
```

Comment:
Can this port instead be sent over NT? That would be much more convenient to the user. Just make a new NT entry.